// linguist-languages/lib/index.js
module.exports = {
  '1C Enterprise': require('../data/1C_20_Enterprise'),
  '2-Dimensional Array': require('../data/2-Dimensional_20_Array'),
  '4D': require('../data/4D'),
  ABAP: require('../data/ABAP'),
  'ABAP CDS': require('../data/ABAP_20_CDS'),
  ABNF: require('../data/ABNF'),
  'AGS Script': require('../data/AGS_20_Script'),
  AIDL: require('../data/AIDL'),
  AL: require('../data/AL'),
  AMPL: require('../data/AMPL'),
  ANTLR: require('../data/ANTLR'),
  'API Blueprint': require('../data/API_20_Blueprint'),
  APL: require('../data/APL'),
  ASL: require('../data/ASL'),
  'ASN.1': require('../data/ASN.1'),
  'ASP.NET': require('../data/ASP.NET'),
  ATS: require('../data/ATS'),
  ActionScript: require('../data/ActionScript'),
  Ada: require('../data/Ada'),
  'Adblock Filter List': require('../data/Adblock_20_Filter_20_List'),
  'Adobe Font Metrics': require('../data/Adobe_20_Font_20_Metrics'),
  Agda: require('../data/Agda'),
  Aiken: require('../data/Aiken'),
  Alloy: require('../data/Alloy'),
  'Alpine Abuild': require('../data/Alpine_20_Abuild'),
  'Altium Designer': require('../data/Altium_20_Designer'),
  AngelScript: require('../data/AngelScript'),
  'Answer Set Programming': require('../data/Answer_20_Set_20_Programming'),
  'Ant Build System': require('../data/Ant_20_Build_20_System'),
  Antlers: require('../data/Antlers'),
  ApacheConf: require('../data/ApacheConf'),
  Apex: require('../data/Apex'),
  'Apollo Guidance Computer': require('../data/Apollo_20_Guidance_20_Computer'),
  AppleScript: require('../data/AppleScript'),
  Arc: require('../data/Arc'),
  AsciiDoc: require('../data/AsciiDoc'),
  AspectJ: require('../data/AspectJ'),
  Assembly: require('../data/Assembly'),
  Astro: require('../data/Astro'),
  Asymptote: require('../data/Asymptote'),
  Augeas: require('../data/Augeas'),
  AutoHotkey: require('../data/AutoHotkey'),
  AutoIt: require('../data/AutoIt'),
  'Avro IDL': require('../data/Avro_20_IDL'),
  Awk: require('../data/Awk'),
  B4X: require('../data/B4X'),
  BASIC: require('../data/BASIC'),
  BQN: require('../data/BQN'),
  Ballerina: require('../data/Ballerina'),
  Batchfile: require('../data/Batchfile'),
  Beef: require('../data/Beef'),
  Befunge: require('../data/Befunge'),
  Berry: require('../data/Berry'),
  BibTeX: require('../data/BibTeX'),
  'BibTeX Style': require('../data/BibTeX_20_Style'),
  Bicep: require('../data/Bicep'),
  Bikeshed: require('../data/Bikeshed'),
  Bison: require('../data/Bison'),
  BitBake: require('../data/BitBake'),
  Blade: require('../data/Blade'),
  BlitzBasic: require('../data/BlitzBasic'),
  BlitzMax: require('../data/BlitzMax'),
  Bluespec: require('../data/Bluespec'),
  'Bluespec BH': require('../data/Bluespec_20_BH'),
  Boo: require('../data/Boo'),
  Boogie: require('../data/Boogie'),
  Brainfuck: require('../data/Brainfuck'),
  BrighterScript: require('../data/BrighterScript'),
  Brightscript: require('../data/Brightscript'),
  Browserslist: require('../data/Browserslist'),
  BuildStream: require('../data/BuildStream'),
  C: require('../data/C'),
  'C#': require('../data/C_23_'),
  'C++': require('../data/C_2b__2b_'),
  'C-ObjDump': require('../data/C-ObjDump'),
  'C2hs Haskell': require('../data/C2hs_20_Haskell'),
  C3: require('../data/C3'),
  'CAP CDS': require('../data/CAP_20_CDS'),
  CIL: require('../data/CIL'),
  CLIPS: require('../data/CLIPS'),
  CMake: require('../data/CMake'),
  COBOL: require('../data/COBOL'),
  CODEOWNERS: require('../data/CODEOWNERS'),
  COLLADA: require('../data/COLLADA'),
  CSON: require('../data/CSON'),
  CSS: require('../data/CSS'),
  CSV: require('../data/CSV'),
  CUE: require('../data/CUE'),
  CWeb: require('../data/CWeb'),
  'Cabal Config': require('../data/Cabal_20_Config'),
  Caddyfile: require('../data/Caddyfile'),
  Cadence: require('../data/Cadence'),
  Cairo: require('../data/Cairo'),
  'Cairo Zero': require('../data/Cairo_20_Zero'),
  CameLIGO: require('../data/CameLIGO'),
  "Cap'n Proto": require('../data/Cap_27_n_20_Proto'),
  Carbon: require('../data/Carbon'),
  CartoCSS: require('../data/CartoCSS'),
  Ceylon: require('../data/Ceylon'),
  Chapel: require('../data/Chapel'),
  Charity: require('../data/Charity'),
  Checksums: require('../data/Checksums'),
  ChucK: require('../data/ChucK'),
  Circom: require('../data/Circom'),
  Cirru: require('../data/Cirru'),
  Clarion: require('../data/Clarion'),
  Clarity: require('../data/Clarity'),
  'Classic ASP': require('../data/Classic_20_ASP'),
  Clean: require('../data/Clean'),
  Click: require('../data/Click'),
  Clojure: require('../data/Clojure'),
  'Closure Templates': require('../data/Closure_20_Templates'),
  'Cloud Firestore Security Rules': require('../data/Cloud_20_Firestore_20_Security_20_Rules'),
  Clue: require('../data/Clue'),
  'CoNLL-U': require('../data/CoNLL-U'),
  CodeQL: require('../data/CodeQL'),
  CoffeeScript: require('../data/CoffeeScript'),
  ColdFusion: require('../data/ColdFusion'),
  'ColdFusion CFC': require('../data/ColdFusion_20_CFC'),
  'Common Lisp': require('../data/Common_20_Lisp'),
  'Common Workflow Language': require('../data/Common_20_Workflow_20_Language'),
  'Component Pascal': require('../data/Component_20_Pascal'),
  Cooklang: require('../data/Cooklang'),
  Cool: require('../data/Cool'),
  'Cpp-ObjDump': require('../data/Cpp-ObjDump'),
  Creole: require('../data/Creole'),
  Crystal: require('../data/Crystal'),
  Csound: require('../data/Csound'),
  'Csound Document': require('../data/Csound_20_Document'),
  'Csound Score': require('../data/Csound_20_Score'),
  Cuda: require('../data/Cuda'),
  'Cue Sheet': require('../data/Cue_20_Sheet'),
  Curry: require('../data/Curry'),
  Cycript: require('../data/Cycript'),
  Cylc: require('../data/Cylc'),
  Cypher: require('../data/Cypher'),
  Cython: require('../data/Cython'),
  D: require('../data/D'),
  'D-ObjDump': require('../data/D-ObjDump'),
  D2: require('../data/D2'),
  'DIGITAL Command Language': require('../data/DIGITAL_20_Command_20_Language'),
  DM: require('../data/DM'),
  'DNS Zone': require('../data/DNS_20_Zone'),
  DTrace: require('../data/DTrace'),
  Dafny: require('../data/Dafny'),
  'Darcs Patch': require('../data/Darcs_20_Patch'),
  Dart: require('../data/Dart'),
  Daslang: require('../data/Daslang'),
  DataWeave: require('../data/DataWeave'),
  'Debian Package Control File': require('../data/Debian_20_Package_20_Control_20_File'),
  DenizenScript: require('../data/DenizenScript'),
  Dhall: require('../data/Dhall'),
  Diff: require('../data/Diff'),
  'DirectX 3D File': require('../data/DirectX_20_3D_20_File'),
  Dockerfile: require('../data/Dockerfile'),
  Dogescript: require('../data/Dogescript'),
  Dotenv: require('../data/Dotenv'),
  Dune: require('../data/Dune'),
  Dylan: require('../data/Dylan'),
  E: require('../data/E'),
  'E-mail': require('../data/E-mail'),
  EBNF: require('../data/EBNF'),
  ECL: require('../data/ECL'),
  ECLiPSe: require('../data/ECLiPSe'),
  EJS: require('../data/EJS'),
  EQ: require('../data/EQ'),
  Eagle: require('../data/Eagle'),
  Earthly: require('../data/Earthly'),
  Easybuild: require('../data/Easybuild'),
  'Ecere Projects': require('../data/Ecere_20_Projects'),
  Ecmarkup: require('../data/Ecmarkup'),
  Edge: require('../data/Edge'),
  EdgeQL: require('../data/EdgeQL'),
  EditorConfig: require('../data/EditorConfig'),
  'Edje Data Collection': require('../data/Edje_20_Data_20_Collection'),
  Eiffel: require('../data/Eiffel'),
  Elixir: require('../data/Elixir'),
  Elm: require('../data/Elm'),
  Elvish: require('../data/Elvish'),
  'Elvish Transcript': require('../data/Elvish_20_Transcript'),
  'Emacs Lisp': require('../data/Emacs_20_Lisp'),
  EmberScript: require('../data/EmberScript'),
  Erlang: require('../data/Erlang'),
  Euphoria: require('../data/Euphoria'),
  'F#': require('../data/F_23_'),
  'F*': require('../data/F_2a_'),
  'FIGlet Font': require('../data/FIGlet_20_Font'),
  FIRRTL: require('../data/FIRRTL'),
  FLUX: require('../data/FLUX'),
  Factor: require('../data/Factor'),
  Fancy: require('../data/Fancy'),
  Fantom: require('../data/Fantom'),
  Faust: require('../data/Faust'),
  Fennel: require('../data/Fennel'),
  'Filebench WML': require('../data/Filebench_20_WML'),
  Filterscript: require('../data/Filterscript'),
  Fluent: require('../data/Fluent'),
  Formatted: require('../data/Formatted'),
  Forth: require('../data/Forth'),
  Fortran: require('../data/Fortran'),
  'Fortran Free Form': require('../data/Fortran_20_Free_20_Form'),
  FreeBASIC: require('../data/FreeBASIC'),
  FreeMarker: require('../data/FreeMarker'),
  Frege: require('../data/Frege'),
  Futhark: require('../data/Futhark'),
  'G-code': require('../data/G-code'),
  GAML: require('../data/GAML'),
  GAMS: require('../data/GAMS'),
  GAP: require('../data/GAP'),
  'GCC Machine Description': require('../data/GCC_20_Machine_20_Description'),
  GDB: require('../data/GDB'),
  GDScript: require('../data/GDScript'),
  GDShader: require('../data/GDShader'),
  GEDCOM: require('../data/GEDCOM'),
  GLSL: require('../data/GLSL'),
  GN: require('../data/GN'),
  GSC: require('../data/GSC'),
  'Game Maker Language': require('../data/Game_20_Maker_20_Language'),
  'Gemfile.lock': require('../data/Gemfile.lock'),
  Gemini: require('../data/Gemini'),
  'Genero 4gl': require('../data/Genero_20_4gl'),
  'Genero per': require('../data/Genero_20_per'),
  Genie: require('../data/Genie'),
  Genshi: require('../data/Genshi'),
  'Gentoo Ebuild': require('../data/Gentoo_20_Ebuild'),
  'Gentoo Eclass': require('../data/Gentoo_20_Eclass'),
  'Gerber Image': require('../data/Gerber_20_Image'),
  'Gettext Catalog': require('../data/Gettext_20_Catalog'),
  Gherkin: require('../data/Gherkin'),
  'Git Attributes': require('../data/Git_20_Attributes'),
  'Git Config': require('../data/Git_20_Config'),
  'Git Revision List': require('../data/Git_20_Revision_20_List'),
  Gleam: require('../data/Gleam'),
  'Glimmer JS': require('../data/Glimmer_20_JS'),
  'Glimmer TS': require('../data/Glimmer_20_TS'),
  Glyph: require('../data/Glyph'),
  'Glyph Bitmap Distribution Format': require('../data/Glyph_20_Bitmap_20_Distribution_20_Format'),
  Gnuplot: require('../data/Gnuplot'),
  Go: require('../data/Go'),
  'Go Checksums': require('../data/Go_20_Checksums'),
  'Go Module': require('../data/Go_20_Module'),
  'Go Workspace': require('../data/Go_20_Workspace'),
  'Godot Resource': require('../data/Godot_20_Resource'),
  Golo: require('../data/Golo'),
  Gosu: require('../data/Gosu'),
  Grace: require('../data/Grace'),
  Gradle: require('../data/Gradle'),
  'Gradle Kotlin DSL': require('../data/Gradle_20_Kotlin_20_DSL'),
  'Grammatical Framework': require('../data/Grammatical_20_Framework'),
  'Graph Modeling Language': require('../data/Graph_20_Modeling_20_Language'),
  GraphQL: require('../data/GraphQL'),
  'Graphviz (DOT)': require('../data/Graphviz_20__28_DOT_29_'),
  Groovy: require('../data/Groovy'),
  'Groovy Server Pages': require('../data/Groovy_20_Server_20_Pages'),
  HAProxy: require('../data/HAProxy'),
  HCL: require('../data/HCL'),
  HIP: require('../data/HIP'),
  HLSL: require('../data/HLSL'),
  HOCON: require('../data/HOCON'),
  HTML: require('../data/HTML'),
  'HTML+ECR': require('../data/HTML_2b_ECR'),
  'HTML+EEX': require('../data/HTML_2b_EEX'),
  'HTML+ERB': require('../data/HTML_2b_ERB'),
  'HTML+PHP': require('../data/HTML_2b_PHP'),
  'HTML+Razor': require('../data/HTML_2b_Razor'),
  HTTP: require('../data/HTTP'),
  HXML: require('../data/HXML'),
  Hack: require('../data/Hack'),
  Haml: require('../data/Haml'),
  Handlebars: require('../data/Handlebars'),
  Harbour: require('../data/Harbour'),
  Hare: require('../data/Hare'),
  Haskell: require('../data/Haskell'),
  Haxe: require('../data/Haxe'),
  HiveQL: require('../data/HiveQL'),
  HolyC: require('../data/HolyC'),
  'Hosts File': require('../data/Hosts_20_File'),
  Hy: require('../data/Hy'),
  HyPhy: require('../data/HyPhy'),
  IDL: require('../data/IDL'),
  'IGOR Pro': require('../data/IGOR_20_Pro'),
  INI: require('../data/INI'),
  'IRC log': require('../data/IRC_20_log'),
  ISPC: require('../data/ISPC'),
  Idris: require('../data/Idris'),
  'Ignore List': require('../data/Ignore_20_List'),
  'ImageJ Macro': require('../data/ImageJ_20_Macro'),
  Imba: require('../data/Imba'),
  'Inform 7': require('../data/Inform_20_7'),
  Ink: require('../data/Ink'),
  'Inno Setup': require('../data/Inno_20_Setup'),
  Io: require('../data/Io'),
  Ioke: require('../data/Ioke'),
  Isabelle: require('../data/Isabelle'),
  'Isabelle ROOT': require('../data/Isabelle_20_ROOT'),
  J: require('../data/J'),
  'JAR Manifest': require('../data/JAR_20_Manifest'),
  JCL: require('../data/JCL'),
  JFlex: require('../data/JFlex'),
  JSON: require('../data/JSON'),
  'JSON with Comments': require('../data/JSON_20_with_20_Comments'),
  JSON5: require('../data/JSON5'),
  JSONLD: require('../data/JSONLD'),
  JSONiq: require('../data/JSONiq'),
  Jai: require('../data/Jai'),
  Janet: require('../data/Janet'),
  Jasmin: require('../data/Jasmin'),
  Java: require('../data/Java'),
  'Java Properties': require('../data/Java_20_Properties'),
  'Java Server Pages': require('../data/Java_20_Server_20_Pages'),
  'Java Template Engine': require('../data/Java_20_Template_20_Engine'),
  JavaScript: require('../data/JavaScript'),
  'JavaScript+ERB': require('../data/JavaScript_2b_ERB'),
  'Jest Snapshot': require('../data/Jest_20_Snapshot'),
  'JetBrains MPS': require('../data/JetBrains_20_MPS'),
  Jinja: require('../data/Jinja'),
  Jison: require('../data/Jison'),
  'Jison Lex': require('../data/Jison_20_Lex'),
  Jolie: require('../data/Jolie'),
  Jsonnet: require('../data/Jsonnet'),
  Julia: require('../data/Julia'),
  'Julia REPL': require('../data/Julia_20_REPL'),
  'Jupyter Notebook': require('../data/Jupyter_20_Notebook'),
  Just: require('../data/Just'),
  KDL: require('../data/KDL'),
  KRL: require('../data/KRL'),
  'Kaitai Struct': require('../data/Kaitai_20_Struct'),
  KakouneScript: require('../data/KakouneScript'),
  KerboScript: require('../data/KerboScript'),
  'KiCad Layout': require('../data/KiCad_20_Layout'),
  'KiCad Legacy Layout': require('../data/KiCad_20_Legacy_20_Layout'),
  'KiCad Schematic': require('../data/KiCad_20_Schematic'),
  Kickstart: require('../data/Kickstart'),
  Kit: require('../data/Kit'),
  Koka: require('../data/Koka'),
  Kotlin: require('../data/Kotlin'),
  Kusto: require('../data/Kusto'),
  LFE: require('../data/LFE'),
  LLVM: require('../data/LLVM'),
  LOLCODE: require('../data/LOLCODE'),
  LSL: require('../data/LSL'),
  'LTspice Symbol': require('../data/LTspice_20_Symbol'),
  LabVIEW: require('../data/LabVIEW'),
  Lark: require('../data/Lark'),
  Lasso: require('../data/Lasso'),
  Latte: require('../data/Latte'),
  Lean: require('../data/Lean'),
  'Lean 4': require('../data/Lean_20_4'),
  Leo: require('../data/Leo'),
  Less: require('../data/Less'),
  Lex: require('../data/Lex'),
  LigoLANG: require('../data/LigoLANG'),
  LilyPond: require('../data/LilyPond'),
  Limbo: require('../data/Limbo'),
  'Linear Programming': require('../data/Linear_20_Programming'),
  'Linker Script': require('../data/Linker_20_Script'),
  'Linux Kernel Module': require('../data/Linux_20_Kernel_20_Module'),
  Liquid: require('../data/Liquid'),
  'Literate Agda': require('../data/Literate_20_Agda'),
  'Literate CoffeeScript': require('../data/Literate_20_CoffeeScript'),
  'Literate Haskell': require('../data/Literate_20_Haskell'),
  'LiveCode Script': require('../data/LiveCode_20_Script'),
  LiveScript: require('../data/LiveScript'),
  Logos: require('../data/Logos'),
  Logtalk: require('../data/Logtalk'),
  LookML: require('../data/LookML'),
  LoomScript: require('../data/LoomScript'),
  Lua: require('../data/Lua'),
  Luau: require('../data/Luau'),
  M: require('../data/M'),
  M3U: require('../data/M3U'),
  M4: require('../data/M4'),
  M4Sugar: require('../data/M4Sugar'),
  MATLAB: require('../data/MATLAB'),
  MAXScript: require('../data/MAXScript'),
  MDX: require('../data/MDX'),
  MLIR: require('../data/MLIR'),
  MQL4: require('../data/MQL4'),
  MQL5: require('../data/MQL5'),
  MTML: require('../data/MTML'),
  MUF: require('../data/MUF'),
  Macaulay2: require('../data/Macaulay2'),
  Makefile: require('../data/Makefile'),
  Mako: require('../data/Mako'),
  Markdown: require('../data/Markdown'),
  Marko: require('../data/Marko'),
  Mask: require('../data/Mask'),
  Mathematica: require('../data/Mathematica'),
  'Maven POM': require('../data/Maven_20_POM'),
  Max: require('../data/Max'),
  Mercury: require('../data/Mercury'),
  Mermaid: require('../data/Mermaid'),
  Meson: require('../data/Meson'),
  Metal: require('../data/Metal'),
  'Microsoft Developer Studio Project': require('../data/Microsoft_20_Developer_20_Studio_20_Project'),
  'Microsoft Visual Studio Solution': require('../data/Microsoft_20_Visual_20_Studio_20_Solution'),
  MiniD: require('../data/MiniD'),
  MiniYAML: require('../data/MiniYAML'),
  MiniZinc: require('../data/MiniZinc'),
  'MiniZinc Data': require('../data/MiniZinc_20_Data'),
  Mint: require('../data/Mint'),
  Mirah: require('../data/Mirah'),
  Modelica: require('../data/Modelica'),
  'Modula-2': require('../data/Modula-2'),
  'Modula-3': require('../data/Modula-3'),
  'Module Management System': require('../data/Module_20_Management_20_System'),
  Mojo: require('../data/Mojo'),
  Monkey: require('../data/Monkey'),
  'Monkey C': require('../data/Monkey_20_C'),
  Moocode: require('../data/Moocode'),
  MoonBit: require('../data/MoonBit'),
  MoonScript: require('../data/MoonScript'),
  Motoko: require('../data/Motoko'),
  'Motorola 68K Assembly': require('../data/Motorola_20_68K_20_Assembly'),
  Move: require('../data/Move'),
  Muse: require('../data/Muse'),
  Mustache: require('../data/Mustache'),
  Myghty: require('../data/Myghty'),
  NASL: require('../data/NASL'),
  NCL: require('../data/NCL'),
  NEON: require('../data/NEON'),
  NL: require('../data/NL'),
  NMODL: require('../data/NMODL'),
  'NPM Config': require('../data/NPM_20_Config'),
  NSIS: require('../data/NSIS'),
  NWScript: require('../data/NWScript'),
  Nasal: require('../data/Nasal'),
  Nearley: require('../data/Nearley'),
  Nemerle: require('../data/Nemerle'),
  NetLinx: require('../data/NetLinx'),
  'NetLinx+ERB': require('../data/NetLinx_2b_ERB'),
  NetLogo: require('../data/NetLogo'),
  NewLisp: require('../data/NewLisp'),
  Nextflow: require('../data/Nextflow'),
  Nginx: require('../data/Nginx'),
  Nickel: require('../data/Nickel'),
  Nim: require('../data/Nim'),
  Ninja: require('../data/Ninja'),
  Nit: require('../data/Nit'),
  Nix: require('../data/Nix'),
  Noir: require('../data/Noir'),
  Nu: require('../data/Nu'),
  NumPy: require('../data/NumPy'),
  Nunjucks: require('../data/Nunjucks'),
  Nushell: require('../data/Nushell'),
  'OASv2-json': require('../data/OASv2-json'),
  'OASv2-yaml': require('../data/OASv2-yaml'),
  'OASv3-json': require('../data/OASv3-json'),
  'OASv3-yaml': require('../data/OASv3-yaml'),
  OCaml: require('../data/OCaml'),
  'OMNeT++ MSG': require('../data/OMNeT_2b__2b__20_MSG'),
  'OMNeT++ NED': require('../data/OMNeT_2b__2b__20_NED'),
  Oberon: require('../data/Oberon'),
  ObjDump: require('../data/ObjDump'),
  'Object Data Instance Notation': require('../data/Object_20_Data_20_Instance_20_Notation'),
  ObjectScript: require('../data/ObjectScript'),
  'Objective-C': require('../data/Objective-C'),
  'Objective-C++': require('../data/Objective-C_2b__2b_'),
  'Objective-J': require('../data/Objective-J'),
  Odin: require('../data/Odin'),
  Omgrofl: require('../data/Omgrofl'),
  Opa: require('../data/Opa'),
  Opal: require('../data/Opal'),
  'Open Policy Agent': require('../data/Open_20_Policy_20_Agent'),
  'OpenAPI Specification v2': require('../data/OpenAPI_20_Specification_20_v2'),
  'OpenAPI Specification v3': require('../data/OpenAPI_20_Specification_20_v3'),
  OpenCL: require('../data/OpenCL'),
  'OpenEdge ABL': require('../data/OpenEdge_20_ABL'),
  OpenQASM: require('../data/OpenQASM'),
  'OpenRC runscript': require('../data/OpenRC_20_runscript'),
  OpenSCAD: require('../data/OpenSCAD'),
  'OpenStep Property List': require('../data/OpenStep_20_Property_20_List'),
  'OpenType Feature File': require('../data/OpenType_20_Feature_20_File'),
  'Option List': require('../data/Option_20_List'),
  Org: require('../data/Org'),
  OverpassQL: require('../data/OverpassQL'),
  Ox: require('../data/Ox'),
  Oxygene: require('../data/Oxygene'),
  Oz: require('../data/Oz'),
  P4: require('../data/P4'),
  PDDL: require('../data/PDDL'),
  'PEG.js': require('../data/PEG.js'),
  PHP: require('../data/PHP'),
  PLSQL: require('../data/PLSQL'),
  PLpgSQL: require('../data/PLpgSQL'),
  'POV-Ray SDL': require('../data/POV-Ray_20_SDL'),
  Pact: require('../data/Pact'),
  Pan: require('../data/Pan'),
  Papyrus: require('../data/Papyrus'),
  Parrot: require('../data/Parrot'),
  'Parrot Assembly': require('../data/Parrot_20_Assembly'),
  'Parrot Internal Representation': require('../data/Parrot_20_Internal_20_Representation'),
  Pascal: require('../data/Pascal'),
  Pawn: require('../data/Pawn'),
  Pep8: require('../data/Pep8'),
  Perl: require('../data/Perl'),
  Pic: require('../data/Pic'),
  Pickle: require('../data/Pickle'),
  PicoLisp: require('../data/PicoLisp'),
  PigLatin: require('../data/PigLatin'),
  Pike: require('../data/Pike'),
  'Pip Requirements': require('../data/Pip_20_Requirements'),
  Pkl: require('../data/Pkl'),
  PlantUML: require('../data/PlantUML'),
  Pod: require('../data/Pod'),
  'Pod 6': require('../data/Pod_20_6'),
  PogoScript: require('../data/PogoScript'),
  Polar: require('../data/Polar'),
  Pony: require('../data/Pony'),
  Portugol: require('../data/Portugol'),
  PostCSS: require('../data/PostCSS'),
  PostScript: require('../data/PostScript'),
  PowerBuilder: require('../data/PowerBuilder'),
  PowerShell: require('../data/PowerShell'),
  Praat: require('../data/Praat'),
  Prisma: require('../data/Prisma'),
  Processing: require('../data/Processing'),
  Procfile: require('../data/Procfile'),
  Proguard: require('../data/Proguard'),
  Prolog: require('../data/Prolog'),
  Promela: require('../data/Promela'),
  'Propeller Spin': require('../data/Propeller_20_Spin'),
  'Protocol Buffer': require('../data/Protocol_20_Buffer'),
  'Protocol Buffer Text Format': require('../data/Protocol_20_Buffer_20_Text_20_Format'),
  'Public Key': require('../data/Public_20_Key'),
  Pug: require('../data/Pug'),
  Puppet: require('../data/Puppet'),
  'Pure Data': require('../data/Pure_20_Data'),
  PureBasic: require('../data/PureBasic'),
  PureScript: require('../data/PureScript'),
  Pyret: require('../data/Pyret'),
  Python: require('../data/Python'),
  'Python console': require('../data/Python_20_console'),
  'Python traceback': require('../data/Python_20_traceback'),
  'Q#': require('../data/Q_23_'),
  QML: require('../data/QML'),
  QMake: require('../data/QMake'),
  'Qt Script': require('../data/Qt_20_Script'),
  Quake: require('../data/Quake'),
  QuakeC: require('../data/QuakeC'),
  QuickBASIC: require('../data/QuickBASIC'),
  R: require('../data/R'),
  RAML: require('../data/RAML'),
  RBS: require('../data/RBS'),
  RDoc: require('../data/RDoc'),
  REALbasic: require('../data/REALbasic'),
  REXX: require('../data/REXX'),
  RMarkdown: require('../data/RMarkdown'),
  RON: require('../data/RON'),
  'ROS Interface': require('../data/ROS_20_Interface'),
  RPC: require('../data/RPC'),
  RPGLE: require('../data/RPGLE'),
  'RPM Spec': require('../data/RPM_20_Spec'),
  RUNOFF: require('../data/RUNOFF'),
  Racket: require('../data/Racket'),
  Ragel: require('../data/Ragel'),
  Raku: require('../data/Raku'),
  Rascal: require('../data/Rascal'),
  'Raw token data': require('../data/Raw_20_token_20_data'),
  ReScript: require('../data/ReScript'),
  'Readline Config': require('../data/Readline_20_Config'),
  Reason: require('../data/Reason'),
  ReasonLIGO: require('../data/ReasonLIGO'),
  Rebol: require('../data/Rebol'),
  'Record Jar': require('../data/Record_20_Jar'),
  Red: require('../data/Red'),
  Redcode: require('../data/Redcode'),
  'Redirect Rules': require('../data/Redirect_20_Rules'),
  'Regular Expression': require('../data/Regular_20_Expression'),
  "Ren'Py": require('../data/Ren_27_Py'),
  RenderScript: require('../data/RenderScript'),
  Rez: require('../data/Rez'),
  'Rich Text Format': require('../data/Rich_20_Text_20_Format'),
  Ring: require('../data/Ring'),
  Riot: require('../data/Riot'),
  RobotFramework: require('../data/RobotFramework'),
  Roc: require('../data/Roc'),
  'Rocq Prover': require('../data/Rocq_20_Prover'),
  Roff: require('../data/Roff'),
  'Roff Manpage': require('../data/Roff_20_Manpage'),
  Rouge: require('../data/Rouge'),
  'RouterOS Script': require('../data/RouterOS_20_Script'),
  Ruby: require('../data/Ruby'),
  Rust: require('../data/Rust'),
  SAS: require('../data/SAS'),
  SCSS: require('../data/SCSS'),
  'SELinux Policy': require('../data/SELinux_20_Policy'),
  SMT: require('../data/SMT'),
  SPARQL: require('../data/SPARQL'),
  SQF: require('../data/SQF'),
  SQL: require('../data/SQL'),
  SQLPL: require('../data/SQLPL'),
  'SRecode Template': require('../data/SRecode_20_Template'),
  'SSH Config': require('../data/SSH_20_Config'),
  STAR: require('../data/STAR'),
  STL: require('../data/STL'),
  STON: require('../data/STON'),
  SVG: require('../data/SVG'),
  SWIG: require('../data/SWIG'),
  Sage: require('../data/Sage'),
  Sail: require('../data/Sail'),
  SaltStack: require('../data/SaltStack'),
  Sass: require('../data/Sass'),
  Scala: require('../data/Scala'),
  Scaml: require('../data/Scaml'),
  Scenic: require('../data/Scenic'),
  Scheme: require('../data/Scheme'),
  Scilab: require('../data/Scilab'),
  Self: require('../data/Self'),
  ShaderLab: require('../data/ShaderLab'),
  Shell: require('../data/Shell'),
  'ShellCheck Config': require('../data/ShellCheck_20_Config'),
  ShellSession: require('../data/ShellSession'),
  Shen: require('../data/Shen'),
  Sieve: require('../data/Sieve'),
  'Simple File Verification': require('../data/Simple_20_File_20_Verification'),
  Singularity: require('../data/Singularity'),
  Slang: require('../data/Slang'),
  Slash: require('../data/Slash'),
  Slice: require('../data/Slice'),
  Slim: require('../data/Slim'),
  Slint: require('../data/Slint'),
  SmPL: require('../data/SmPL'),
  Smali: require('../data/Smali'),
  Smalltalk: require('../data/Smalltalk'),
  Smarty: require('../data/Smarty'),
  Smithy: require('../data/Smithy'),
  Snakemake: require('../data/Snakemake'),
  Solidity: require('../data/Solidity'),
  Soong: require('../data/Soong'),
  SourcePawn: require('../data/SourcePawn'),
  'Spline Font Database': require('../data/Spline_20_Font_20_Database'),
  Squirrel: require('../data/Squirrel'),
  Stan: require('../data/Stan'),
  'Standard ML': require('../data/Standard_20_ML'),
  Starlark: require('../data/Starlark'),
  Stata: require('../data/Stata'),
  StringTemplate: require('../data/StringTemplate'),
  Stylus: require('../data/Stylus'),
  'SubRip Text': require('../data/SubRip_20_Text'),
  SugarSS: require('../data/SugarSS'),
  SuperCollider: require('../data/SuperCollider'),
  'Survex data': require('../data/Survex_20_data'),
  Svelte: require('../data/Svelte'),
  Sway: require('../data/Sway'),
  Sweave: require('../data/Sweave'),
  Swift: require('../data/Swift'),
  SystemVerilog: require('../data/SystemVerilog'),
  'TI Program': require('../data/TI_20_Program'),
  'TL-Verilog': require('../data/TL-Verilog'),
  TLA: require('../data/TLA'),
  TOML: require('../data/TOML'),
  'TSPLIB data': require('../data/TSPLIB_20_data'),
  TSQL: require('../data/TSQL'),
  TSV: require('../data/TSV'),
  TSX: require('../data/TSX'),
  TXL: require('../data/TXL'),
  Tact: require('../data/Tact'),
  Talon: require('../data/Talon'),
  Tcl: require('../data/Tcl'),
  Tcsh: require('../data/Tcsh'),
  TeX: require('../data/TeX'),
  Tea: require('../data/Tea'),
  Terra: require('../data/Terra'),
  'Terraform Template': require('../data/Terraform_20_Template'),
  Texinfo: require('../data/Texinfo'),
  Text: require('../data/Text'),
  TextGrid: require('../data/TextGrid'),
  'TextMate Properties': require('../data/TextMate_20_Properties'),
  Textile: require('../data/Textile'),
  Thrift: require('../data/Thrift'),
  Toit: require('../data/Toit'),
  'Tor Config': require('../data/Tor_20_Config'),
  'Tree-sitter Query': require('../data/Tree-sitter_20_Query'),
  Turing: require('../data/Turing'),
  Turtle: require('../data/Turtle'),
  Twig: require('../data/Twig'),
  'Type Language': require('../data/Type_20_Language'),
  TypeScript: require('../data/TypeScript'),
  TypeSpec: require('../data/TypeSpec'),
  Typst: require('../data/Typst'),
  'Unified Parallel C': require('../data/Unified_20_Parallel_20_C'),
  'Unity3D Asset': require('../data/Unity3D_20_Asset'),
  'Unix Assembly': require('../data/Unix_20_Assembly'),
  Uno: require('../data/Uno'),
  UnrealScript: require('../data/UnrealScript'),
  'Untyped Plutus Core': require('../data/Untyped_20_Plutus_20_Core'),
  UrWeb: require('../data/UrWeb'),
  V: require('../data/V'),
  VBA: require('../data/VBA'),
  VBScript: require('../data/VBScript'),
  VCL: require('../data/VCL'),
  VHDL: require('../data/VHDL'),
  Vala: require('../data/Vala'),
  'Valve Data Format': require('../data/Valve_20_Data_20_Format'),
  'Velocity Template Language': require('../data/Velocity_20_Template_20_Language'),
  Vento: require('../data/Vento'),
  Verilog: require('../data/Verilog'),
  'Vim Help File': require('../data/Vim_20_Help_20_File'),
  'Vim Script': require('../data/Vim_20_Script'),
  'Vim Snippet': require('../data/Vim_20_Snippet'),
  'Visual Basic .NET': require('../data/Visual_20_Basic_20_.NET'),
  'Visual Basic 6.0': require('../data/Visual_20_Basic_20_6.0'),
  Volt: require('../data/Volt'),
  Vue: require('../data/Vue'),
  Vyper: require('../data/Vyper'),
  WDL: require('../data/WDL'),
  WGSL: require('../data/WGSL'),
  'Wavefront Material': require('../data/Wavefront_20_Material'),
  'Wavefront Object': require('../data/Wavefront_20_Object'),
  'Web Ontology Language': require('../data/Web_20_Ontology_20_Language'),
  WebAssembly: require('../data/WebAssembly'),
  'WebAssembly Interface Type': require('../data/WebAssembly_20_Interface_20_Type'),
  WebIDL: require('../data/WebIDL'),
  WebVTT: require('../data/WebVTT'),
  'Wget Config': require('../data/Wget_20_Config'),
  Whiley: require('../data/Whiley'),
  Wikitext: require('../data/Wikitext'),
  'Win32 Message File': require('../data/Win32_20_Message_20_File'),
  'Windows Registry Entries': require('../data/Windows_20_Registry_20_Entries'),
  'Witcher Script': require('../data/Witcher_20_Script'),
  Wollok: require('../data/Wollok'),
  'World of Warcraft Addon Data': require('../data/World_20_of_20_Warcraft_20_Addon_20_Data'),
  Wren: require('../data/Wren'),
  'X BitMap': require('../data/X_20_BitMap'),
  'X Font Directory Index': require('../data/X_20_Font_20_Directory_20_Index'),
  'X PixMap': require('../data/X_20_PixMap'),
  X10: require('../data/X10'),
  XC: require('../data/XC'),
  XCompose: require('../data/XCompose'),
  XML: require('../data/XML'),
  'XML Property List': require('../data/XML_20_Property_20_List'),
  XPages: require('../data/XPages'),
  XProc: require('../data/XProc'),
  XQuery: require('../data/XQuery'),
  XS: require('../data/XS'),
  XSLT: require('../data/XSLT'),
  Xmake: require('../data/Xmake'),
  Xojo: require('../data/Xojo'),
  Xonsh: require('../data/Xonsh'),
  Xtend: require('../data/Xtend'),
  YAML: require('../data/YAML'),
  YANG: require('../data/YANG'),
  YARA: require('../data/YARA'),
  YASnippet: require('../data/YASnippet'),
  Yacc: require('../data/Yacc'),
  Yul: require('../data/Yul'),
  ZAP: require('../data/ZAP'),
  ZIL: require('../data/ZIL'),
  Zeek: require('../data/Zeek'),
  ZenScript: require('../data/ZenScript'),
  Zephir: require('../data/Zephir'),
  Zig: require('../data/Zig'),
  Zimpl: require('../data/Zimpl'),
  Zmodel: require('../data/Zmodel'),
  'cURL Config': require('../data/cURL_20_Config'),
  crontab: require('../data/crontab'),
  desktop: require('../data/desktop'),
  dircolors: require('../data/dircolors'),
  eC: require('../data/eC'),
  edn: require('../data/edn'),
  fish: require('../data/fish'),
  hoon: require('../data/hoon'),
  iCalendar: require('../data/iCalendar'),
  jq: require('../data/jq'),
  kvlang: require('../data/kvlang'),
  'mIRC Script': require('../data/mIRC_20_Script'),
  mcfunction: require('../data/mcfunction'),
  mdsvex: require('../data/mdsvex'),
  mupad: require('../data/mupad'),
  nanorc: require('../data/nanorc'),
  nesC: require('../data/nesC'),
  ooc: require('../data/ooc'),
  q: require('../data/q'),
  reStructuredText: require('../data/reStructuredText'),
  'robots.txt': require('../data/robots.txt'),
  sed: require('../data/sed'),
  templ: require('../data/templ'),
  vCard: require('../data/vCard'),
  wisp: require('../data/wisp'),
  xBase: require('../data/xBase'),
}
